### ISSUE TITLE
Extend expect to not have to use ugly response and improve output

### ISSUE THREAD

**Issue by DummettTech** [DummettTech/No-A11y#3]
Just throws all the data out, ugly.

- better looking code `expect(result).toPass` (?)
- better output (selector, violation type, what it's coming from e.g wcagaa)
